includes/Admin.php

```diff
@@ -145,6 +145,14 @@ public function enqueue_scripts() {
 		}
 
 		wp_enqueue_script( 'wc-facebook-google-product-category-fields', facebook_for_woocommerce()->get_plugin_url() . '/assets/js/admin/google-product-category-fields.min.js', [ 'jquery' ], \WC_Facebookcommerce::PLUGIN_VERSION );
+
+		wp_localize_script( 'wc-facebook-google-product-category-fields', 'facebook_for_woocommerce_google_product_category', [
+			'i18n' => [
+				'top_level_placeholder'    => __( 'Search main categories...', 'facebook-for-woocommerce' ),
+				'second_level_placeholder' => __( 'Choose a main category', 'facebook-for-woocommerce' ),
+				'general_placeholder'      => __( 'Choose a parent category', 'facebook-for-woocommerce' ),
```

Comment:
maybe these key names could be more unambiguous, for example that are about a dropdown placeholder?

Reply:
Like `top_level_dropdown_placeholder`? I can added once we confirm I got the idea right :)